aea/protocols/fipa/fipa.proto

```diff
@@ -23,6 +23,14 @@ message FIPAMessage{
         bytes bytes = 1;
     }
 
+     message Accept_W_Address{
```

Comment:
there seems to be a whitespace before `message`

Reply:
To be consistent, I suggest we change from `Accept_W_Address` to `AcceptWAddress` and similarly, `MatchAcceptWAddress`

Reply:
Only here in proto